App.js

```diff
@@ -63,7 +55,9 @@ class App extends Component {
 
         return (
             <Provider store={store}>
-                <MainNavigator/>
+                <PersistGate loading={<AppLoading/>} persistor={persistor}>
+                    <MainNavigator/>
+                </PersistGate>
             </Provider>
```

Comment:
`PersistGate` is a nice component that comes with `redux-persist` which delays the rendering in `React` until all of the rehydration is complete. Instead of the default `null` value, I passed the `AppLoading` component from `expo` so it shows our splash screen while this booting up is happening. It knows it's done when the `persistor` that we passed is done doing its thing.

Reply:
that's really handy.